### PR TITLE
core: fix generate ObjectId length less than 24. (#6991)

### PR DESCRIPTION
**Pull request by dyw770**

### CODE REVIEW

**Comment by dyw770** [comment]
#6991

File: packages/core/src/utils/object-id.ts [hunk 1]
```diff
@@ -19,8 +19,16 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 const MACHINE_ID = Math.floor(Math.random() * 0xffffff);
 const pid = Math.floor(Math.random() * 100000) % 0xffff;
+
+let machineIdStr = MACHINE_ID.toString(16);
+if (machineIdStr.length !== 6) machineIdStr = machineIdStr.padStart(6, "0");
```
Comment (thecodrr) [inline packages/core/src/utils/object-id.ts:24]:
```suggestion
let machineIdStr = MACHINE_ID.toString(16).padStart(6, "0");
```

File: packages/core/src/utils/object-id.ts [hunk 2]
```diff
@@ -19,8 +19,16 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
+let pidStr =  pid.toString(16);
+if (pidStr.length !== 4) pidStr = pidStr.padStart(4, "0");
```
Comment (thecodrr) [inline packages/core/src/utils/object-id.ts:26]:
```suggestion
let pidStr =  pid.toString(16).padStart(4, "0");
```

File: packages/core/src/utils/object-id.ts [hunk 3]
```diff
@@ -19,8 +19,14 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 const MACHINE_ID = Math.floor(Math.random() * 0xffffff);
 const pid = Math.floor(Math.random() * 100000) % 0xffff;
+
+let machineIdStr = MACHINE_ID.toString(16).padStart(6, "0");
+let pidStr =  pid.toString(16).padStart(4, "0");
+
+const PROCESS_UNIQUE = machineIdStr + pidStr;
+
```
Comment (thecodrr) [inline packages/core/src/utils/object-id.ts:27]:
```suggestion
const PROCESS_UNIQUE = MACHINE_ID.toString(16).padStart(6, "0") + pid.toString(16).padStart(4, "0");
```

File: packages/core/src/utils/object-id.ts [hunk 4]
```diff
@@ -19,8 +19,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 const MACHINE_ID = Math.floor(Math.random() * 0xffffff);
 const pid = Math.floor(Math.random() * 100000) % 0xffff;
+
+const PROCESS_UNIQUE = MACHINE_ID.toString(16).padStart(6, "0") + pid.toString(16).padStart(4, "0");
```
Comment (thecodrr) [inline packages/core/src/utils/object-id.ts:23]:
```suggestion
const PROCESS_UNIQUE = MACHINE_ID.toString(16).padStart(6, "0") + pid.toString(16).padStart(4, "0");
```

**Review by thecodrr — APPROVED** [review]

**Pull request merged**